backend/src/common/packages/http/enums/http-code.enum.ts

```diff
@@ -0,0 +1,18 @@
+const HttpCode = {
```

Comment:
it is related to http package for frontend, what if someone decide to change it and it will affect our package. also, I made some changes in my version for our case.

Reply:
This will be out of our quality criteria (A11). But I'm not good in this field.

Reply:
it possible, but should be renamed to httpBackend package

Reply:
Please use shared enum for this, because